### ISSUE TITLE
[qt5cpp] redefinition of 'output'

### ISSUE THREAD

**Issue by MartinDelille** [swagger-api/swagger-codegen#7785]
##### Description

When I try to compile the generated code from the Pet store API, it fails compiling with this error:

```
../../../../tests/swagger/petstore/gen/client/SWGStoreApi.cpp:123:28: error: redefinition of 'output'
    QMap<QString, qint32>* output = new QMap<QString, qint32>();
                           ^
../../../../tests/swagger/petstore/gen/client/SWGStoreApi.cpp:122:28: note: previous definition is here
    QMap<QString, qint32>* output;  // TODO add primitive output support
                           ^
1 error generated.
make: *** [SWGStoreApi.o] Error 1
18:42:35: The process "/usr/bin/make" exited with code 2.
```

##### Swagger-codegen version

2.3.1

##### Swagger declaration file content or url

https://gist.github.com/MartinDelille/c358209832ed023c1cd7ddb308db0099

##### Command line used for generation

    $ swagger-codegen generate -i api.yml -l qt5cpp

##### Steps to reproduce

- Generate the code from the pet store example (link in the gist above)
- Create a Qt project including all the generated files
- Compile it

##### Related issues/PRs

Not that I know.

##### Suggest a fix/enhancement

Hard to step into the code like that, but the strange thing is that there is no problem on the code sample: https://github.com/swagger-api/swagger-codegen/blob/master/samples/client/petstore/qt5cpp/client/SWGStoreApi.cpp#L122

On my generated code I have the following lines in SWGStoreApi.cpp starting line 122:

```cpp
    QMap<QString, qint32>* output;  // TODO add primitive output support
    QMap<QString, qint32>* output = new QMap<QString, qint32>();
    QString json(worker->response);
    QByteArray array (json.toStdString().c_str());
    QJsonDocument doc = QJsonDocument::fromJson(array);
    QJsonObject obj = doc.object();
```

This TODO seems to come from this line: https://github.com/swagger-api/swagger-codegen/blob/12f3661d6fc56f0c7a828c6f7f751f3e40ebfe22/modules/swagger-codegen/src/main/resources/qt5cpp/api-body.mustache#L166

**Comment by etherealjoy** [comment]
@MartinDelille 
Yes, I have seen this bug before. The template is not handling it correctly for now.
However, code sample does not have a API description triggering this code path.

**Comment by MartinDelille** [comment]
The bug is fixed with 2.4.0.
